src/pgrn-query-expand.c

```diff
@@ -502,7 +503,19 @@ pgroonga_query_expand(PG_FUNCTION_ARGS)
 	}
 	tableOID = DatumGetObjectId(tableOIDDatum);
 	currentData.table = RelationIdGetRelation(tableOID);
-
+#ifdef PGRN_HAVE_RELKIND_HAS_TABLE_AM
+	if(!RELKIND_HAS_TABLE_AM(currentData.table->rd_rel->relkind))
```

Comment:
Could you use the same style?

```suggestion
	if (!RELKIND_HAS_TABLE_AM(currentData.table->rd_rel->relkind))
```

Reply:
Could you use `RelationGetForm()` instead of `->rd_rel`?